### ISSUE TITLE
After completing Sign In: accessNotConfigured popup

### ISSUE THREAD

**Issue by gasull** [anxdpanic/plugin.video.youtube#348]
### Context

Please provide any relevant information about your setup

* Add-on Version: 6.8.20+matrix.1
* Kodi Version: 2:19.4-2~bullseye
* Kodi GUI Language: English
* Operating System: Raspberry OS - Debian GNU/Linux 11
* Operating System Language: English

------

### Expected Behavior

After signing in, clicking on Popular right now should display the popular videos

------

### Current Behavior

I complete the signin, going to https://google.com/device twice, and completing the steps successfully.  Afterwards, I click on "Popular right now", and I get a popup saying accessNotConfigured, despite having API keys configured and having completed the sign up.

------

### Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. Sign in
2. Click on Popular right now.
3. 

------

### Log

2022-11-04 15:34:12.421 T:3695    ERROR <general>: GetDirectory - Error getting plugin://plugin.video.youtube/special/popular_right_now/
2022-11-04 15:34:12.421 T:3693     INFO <general>: CPythonInvoker(6, /home/akeekeih/.kodi/addons/plugin.video.youtube/resources/lib/default.py): script successfully run
2022-11-04 15:34:12.454 T:3618    ERROR <general>: CGUIMediaWindow::GetDirectory(plugin://plugin.video.youtube/special/popular_right_now/) failed
2022-11-04 15:34:16.684 T:3693     INFO <general>: initializing python engine.
2022-11-04 15:34:16.688 T:3693     INFO <general>: [plugin.video.youtube] Running: YouTube (6.8.20+matrix.1) on Matrix (Kodi-19.4) with Python 3.9.2
                                                        Path: /special/popular_right_now/
                                                        Params: {}
2022-11-04 15:34:16.900 T:3693    ERROR <general>: [plugin.video.youtube] Error reason: |accessNotConfigured| with message: |YouTube Data API v3 has not been used in project 804514510728 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/youtube.googleapis.com/overview?project=804514510728 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.|

------

### Additional Information

Please provide any additional information that may be helpful.

------

**Comment by derMart** [comment]
One needs to go to the link (https://console.developers.google.com/apis/api/youtube.googleapis.com/overview?project=<your_dev_project_id>) in the kodi log (under storage/.kodi/temp/kodi.log) and enable the youtube api v3 for the created project.  
An alternative is to visit https://console.cloud.google.com/apis/dashboard (for your google dev project) and click on 'enable apis and services' then search for 'youtube data api v3' and enable it.
In short: The youtube data api needs to be enabled for the project.
